docs/introduction.md

```diff
@@ -57,12 +57,14 @@ The following functions are provided:
 - Arithmetic, comparison, shifting, and validation operations for packed decimal data.
 - Conversion operations between different binary coded decimal and Java binary types.
 - Marshalling operations: marshalling and unmarshalling Java binary types, such as `short`, `int`, `long`, `float`, and `double`, to and from byte arrays.
+- External decimals support with all four sign configurations: sign embedded trailing, sign embedded leading, sign separate trailing, and sign separate leading. It also accommodates sign embedded trailing with spaces.
 
 You can gain a number of benefits by using the APIs provided:
 
 - Improved application performance by avoiding object creation and intermediate processing, which can also put pressure on the Java heap.
 - Hardware acceleration by automatically exploiting available hardware features on specific platforms.
 - Platform independence for applications that are developed to take advantage of Data Access Acceleration.
+- Validate the sign and digits of a given external decimal input before operating on the data.
```

Comment:
we should mention packed decimal as well here

Reply:
Based on https://github.com/eclipse-openj9/openj9-docs/pull/1506#discussion_r2053290417, this is no longer needed